src/form/validators/emailValidation/emailValidation.ts

```diff
@@ -0,0 +1,14 @@
+export const emailValidation = (maxLength: number) => ({
```

Comment:
Właściwie to nie jest `emailValidation` tylko `requiredMaxLengthEmailValidation` :) Dobrze by to było rodzielić na trzy różne walidatory

Reply:
Hmmm, zrobiłem to ale nie do końca czaję sens tego rozbicia.